### ISSUE TITLE
未牵引的情况下为啥IoI指标也会逐轮次升高呢？

### ISSUE THREAD

**Issue by DayUp1130** [GabyUSTC/IPG-Rec#1]
您好，有个问题，在不牵引的情况下，直接使用next_item_prediction即
` def next_item_prediction(self, log_seqs, interacted):
        log_feats = self.log2feats(log_seqs)
        final_feat = log_feats[:, -1, :]
        logits = torch.matmul(final_feat, item_embs.T)
        logits[interacted[:, 1:] == 1] = float('-inf')
        _, next_items = logits.max(dim=1)
       return nex_items 
`
这种情况下产生的recommendation为啥IoI(increase of interest)指标也会逐轮次提升呢？不是生成的next item完全跟target item无关吗？是我理解有误，代码编写不对导致的吗？
期待您的解答。

**Comment by GabyUSTC** [comment]
next_item_prediction中有一个参数alpha，如果这个数大于0的话也是会牵引的，如果设置成0（完全使用SASRec），在我的结果中应该IoI是略小于0

**Comment by DayUp1130** [comment]
邮件已经收到，尽快给您回复~
